src/libusb.c

```diff
@@ -121,6 +85,12 @@ libusb_find_more_about_nic(device_t *device)
   int totalEndpoints = 0;
   struct usb_device *libusb_device = libusb_findDevice(device->vendorid, device->deviceid);	
 
+  if (!libusb_device)
+  {
+    xd_log(LOG_WARNING, "Unable to find device with vendor ID %d and device ID %d. Was it removed?", device->vendorid, device->deviceid);
```

Comment:
4 hex digits (%04x) for VID/PID is how they are more typically printed.  I suggest you change to that.